doc/chomp_interface/scripts/collision_scene_test_1.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+import rospy
+from moveit_commander import RobotCommander, PlanningSceneInterface
+import geometry_msgs.msg
+import time
+
+
+class CreateScene(object):
```

Comment:
better class name.

Reply:
changed class name to ``CreateClutteredScene`` should I keep the filename same as the class name or is this fine as is, filename = ``collision_scene_example_cluttered.py``?